### PR TITLE
Keep only services needed from kubeflow

### PR DESCRIPTION
**Pull request by GerardoGR**
Services removed:
- Knative
- Volumes and Notebooks webapp
- Training operator

### CODE REVIEW

File: components/kubeflow-notebooks/kustomize/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    spec:
+      serviceAccountName: fl-notebook
+      containers:
+        - env:
+          - name: NB_PREFIX
+            value: /jupyter
+          - name: KF_PIPELINES_SA_TOKEN_PATH
+            value: /var/run/secrets/kubeflow/pipelines/token
+          # TODO: Parametrize
+          image: localhost:5000/katulu-fl-jupyter-full:1657174761
```
Comment (orlandohohmeier) [inline components/kubeflow-notebooks/kustomize/deployment.yaml]: This needs to be parameterized as the TODO said before we can merge this to install the image from the correct registry.

**Review by orlandohohmeier — CHANGES_REQUESTED** [review]

File: components/kubeflow-notebooks/kustomize/deployment.yaml (hunk 1 above)
Comment (GerardoGR) [reply components/kubeflow-notebooks/kustomize/deployment.yaml]: Thanks for the reminder. Removed!

File: components/kubeflow-notebooks/Makefile [hunk 2]
```diff
@@ -18,16 +20,22 @@ build:
...
 	mkdir -p dist
 	cp ../kubeflow-pipeline/python/dist/fl_suite-${PIP_VERSION}-py3-none-any.whl dist/
 	sed -e "s/\\\$$\\\$$container-tag\\\$$\\\$$/${PIP_VERSION}/g" Dockerfile.tpl > dist/Dockerfile
 	sed -e "s/\\\$$\\\$$container-tag\\\$$\\\$$/${PIP_VERSION}/g" requirements.tpl.txt > dist/requirements.txt
-	DOCKER_BUILDKIT=1 docker build -t ${MAKEVAR_REGISTRY}/katulu-fl-jupyter-full:${CONTAINER_TAG} dist
+	DOCKER_BUILDKIT=1 docker build -t ${IMAGE_URL} dist
 .PHONY: dist
 
+manifests:
+	@cd kustomize/base/ && \
```
Comment (orlandohohmeier) [inline components/kubeflow-notebooks/Makefile]: Let's have a component level kustomize template instead to follow the general component layout

**Review by orlandohohmeier — CHANGES_REQUESTED** [review]

File: components/kubeflow-notebooks/Makefile (hunk 2 above)
Comment (GerardoGR) [reply components/kubeflow-notebooks/Makefile]: Good point. I've amended the PR.

**Review by orlandohohmeier — APPROVED** [review]

**Pull request merged**